app/views/layouts/application.slim

```diff
@@ -9,7 +9,10 @@ html
     = render partial: 'shared/rollbarjs'
     = javascript_pack_tag 'application'
     = javascript_include_tag 'application'
+    = javascript_include_tag '/assets/toolkit.min.js'
+    = javascript_include_tag 'https://unpkg.com/tippy.js/dist/tippy.all.min.js'
```

Comment:
Is there a reason as to not have this be managed by yarn and to load it from the URL specifically?

Reply:
Didn't know how, think I got it now though. Should be fixed